week1/index.html

```diff
@@ -0,0 +1,132 @@
+<!DOCTYPE html>
+
+<head>
+    <meta charset="utf-8">
+    <title>Make Your Home Great Again,Home Office</title>
+    <meta name="author" content="Ekip Kalir">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" href="main.css">
+    <script src="https://kit.fontawesome.com/f32ec08315.js" crossorigin="anonymous"></script>
+</head>
+
+<body>
+    <div class="nav-bar-top">
+        <header>
+            <ul>
+                <li><a href="#company-details">BlogX</a></li>
+                <li><a href="#articles-whole">Content</a></li>
+                <li><a href="#contact-details">Contact</a></li>
+                <li style="float:right"><a class="active" href="#summary">About</a></li>
+            </ul>
+        </header>
+        <br>
+        <br>
+        <div class="brand-logo">
+            <figure>
+                <img src="blogx-logo.png" style="box-shadow: 5px 10px 8px #2c2c2c" alt="BlogX Logo Looking">
+                <figcaption style="font-size:30px;columns: #f5f4f4;; text-shadow: 5px 5px 10px #2c2e3d">
+                    <strong><a href="#logo">BlogX,Your Blog Journey</strong></a></figcaption>
```

Comment:
Tags are slightly muddled here; make sure you open and close tags in the correct order. Never close an outer tag before the inner tag has been closed.

Reply:
Done.